src/bin/lib/parson/parson.c

```diff
@@ -2,7 +2,7 @@
  SPDX-License-Identifier: MIT
 
  Parson 1.5.3 (https://github.com/kgabis/parson)
- Copyright (c) 2012 - 2023 Krzysztof Gabis
+ Copyright (c) 2012 - 2025 Krzysztof Gabis
```

Comment:
Please see about updating our copy of the parson lib from the upstream, not just the copyright line.

Reply:
Ah, sorry, I must have been on autopilot. I'll take a look at this in the next days.

Reply:
No changes in `parson.c`, dropping this.